diffrax/_solver/langevin_srk.py

```diff
@@ -0,0 +1,325 @@
+import abc
+from typing import Any, Callable, Generic, TypeVar
+
+import equinox as eqx
+import equinox.internal as eqxi
+import jax.lax as lax
+import jax.numpy as jnp
+import jax.tree_util as jtu
+from equinox import AbstractVar
+from jax import vmap
+from jax._src.tree_util import PyTreeDef
```

Comment:
I don't think this requires a `_src` import.

(If nothing else it's available in `jaxtyping`.)

Reply:
This one is on PyCharm (and me not checking 😅)